src/enums/animated-mime-types.enum.ts

```diff
@@ -1,6 +1,6 @@
 export enum NonStaticMimeTypes {
   VIDEO = 'video/mp4',
-  AUDIO = 'audio/mpeg',
+  AUDIO = 'audio',
```

Comment:
if you noticed some more mime types besides "mpeg", just add them to this enum, like "AUDIO_MPEG = audio/mpeg..."